functions-methods.js

```diff
@@ -33,4 +85,36 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+// Requirements
+// Ask email-adress
+// Make requirements for email-adress:
+//     - has to include an "@"
+//     - does not have ","
+//     - does not have "." at the end
+// Return true or false
+
+//Steps:
+// X Declare function
+// X Set parameters
+// X Set requirements for email-adress
+// X Check if arguments fit requirements
+// X Return true or false
+
+function checkEmailValidity(emailAdress) {
+    const includesAt = emailAdress.includes("@"); //Look for "@"
+    const noComma = emailAdress.includes(","); // Look for ","
+    const noPeriodLastCharacter = emailAdress.substring(emailAdress.length - 1).includes(".") // Look for last character and if it's "
+    if (includesAt === true && noComma !== true && noPeriodLastCharacter !== true) {
```

Comment:
`includesAt === true` is hetzelfde als `includesAt`. Dit kan dus korter: `if (includesAt && noComma && noPeriodLastCharacter) {